index.html

```diff
@@ -4,27 +4,158 @@
+
+
+          <div id="video-info">
+            <h2>What is VCS? (Git-SCM) • Git Basics #1</h2>
+            <div class="below-video">
+              <div class="user-info">
+                <img src="https://yt3.ggpht.com/a/AGF-l7_gYj8aaqXkkS8JDFC4bzX4EUU7vzdKnQdLRA=s288-c-k-c0xffffffff-no-rj-mo" class="user-pic" />
+                <div>
+                  <b class="user-name">The Odin Project</b>
+                  <div>
+                    <button><i class="fab fa-youtube"></i> <b>Subscribe</b></button>
+                    <label>799</label>
+                  </div>
+                </div>
+              </div>
+              <label for=""><i>50,106</i> views</label>
+            </div>
+            <div class="video-icons">
+              <div class="right">
+                <i id="add" class="fas fa-plus"></i>
+                <i id="share" class="fas fa-share"></i>
+                <i id="more" class="fas fa-ellipsis-h"></i>
+              </div>
+              <div class="left">
+                <i id="like" class="fas fa-thumbs-up"><span> 768</span></i>
+                <i id="dislike" class="fas fa-thumbs-down"><span> 12</span></i>
+              </div>
+            </div>
+          </div>
+
+          <div class="description">
+            <h4>Published on Feb 27, 2014</h4>
+            <p>
+              How do you contribute to open source? In this video, I'll describe
+              the easiest way to make a simple contribution to an open source
+              project -- without using the command line, a text editor, "forking a
+              repo", or even touching Git outside of Github. I'll also describe
+              what kinds of contributions open source software projects are
+              typically looking for. As an example, I use submitting a student
+              solution to a web development exercise from The Odin Project, a free
+              online curriculum for learning web development with Ruby on Rails.
+              You can find more information about that project at
+              http://theodinproject.com.
+            </p>
+            <div class="show-more">SHOW MORE</div>
+          </div>
+  
+          <div class="comments">
+            <div><i class="fas fa-spinner fa-pulse"></i><label> loading...</label></div>
+          </div>
+        </div>
+    
+
+        <aside>
+          <div class="up-next">
+            <label>Up next</label>
+            <form>
+              <label>Autoplay</label><i class="fas fa-info-circle"></i><i class="fas fa-toggle-on"></i>
+            </form>
+          </div>
+  
+          <ul>
+            <li class="side-videos">
+              <img src="https://i.ytimg.com/vi/ReE87Wl6c4I/maxresdefault.jpg" alt="" class="thumbnail" />
+              <div>
+                <h3>How To Make YouTube Thumbnails For Free Online</h3>
+                <label for="" class="thumb-info"></label>
+                <label for="" class="thumb-info"></label>
+              </div>
```

Comment:
Nice! :+1: I see what you're trying to achieve, just one small snag though, a div is higher than a list item in heirachy so semantically, this may not be the greatest choice, give it a second look!